uni/lib/model/entities/reference.dart

```diff
@@ -0,0 +1,21 @@
+class Reference {
+  late final String description;
+  late final DateTime limitDate;
+  late final int entity;
+  late final int reference;
+  late final double amount;
```

Comment:
Does this need to be late?

Reply:
No, you're right! I don't know why I made them late tbh